### PR TITLE
fix: move `beforeHandler` from schema in home page example

### PR DESCRIPTION
**Pull request by olehdevua**

### CODE REVIEW

File: src/website/layouts/home.html [hunk 1]
```diff
@@ -126,11 +126,11 @@ <h2>Request/Response validation and hooks</h2>
...
     },
-    // this function is executed for every request before the handler is executed
-    beforeHandler: (request, reply) => {
-      // E.g. check authentication
-    }
   },
+  // this function is executed for every request before the handler is executed
+  beforeHandler: (request, reply) => {
+    // E.g. check authentication
+  }
```
Comment (jsumners) [inline src/website/layouts/home.html]: Missing a `,`.

**Review by delvedor — APPROVED** [review]
LGTM with @jsumners's nit.

**Comment by olehdevua** [comment]
@delvedor now everything is ok

**Comment by lmammino** [comment]
@delvedor @jsumners, should we get this merged?

**Review by jsumners — APPROVED** [review]
LGTM

File: src/website/layouts/home.html [hunk 2]
```diff
@@ -126,10 +126,10 @@ <h2>Request/Response validation and hooks</h2>
         }
       }
     },
-    // this function is executed for every request before the handler is executed
-    beforeHandler: (request, reply) => {
-      // E.g. check authentication
-    }
+  },
+  // this function is executed for every request before the handler is executed
+  beforeHandler: (request, reply) => {
```
Comment (delvedor) [inline src/website/layouts/home.html]: I've just noticed that here is missing the `done` callback :P
https://github.com/fastify/fastify/blob/master/docs/Hooks.md#beforehandler

**Comment by olehdevua** [comment]
@delvedor @jsumners , I added `done` callback which `beforeHandler` is supposed to have.

**Review by delvedor — APPROVED** [review]
LGTM

**Review by jsumners — APPROVED** [review]
LGTM

Sorry for not catching that the first time.

**Pull request merged**